content/influxdb/v2.0/tools/grafana.md

```diff
@@ -69,8 +69,64 @@ configure your InfluxDB connection:
 {{% tab-content %}}
 ## Configure Grafana to use InfluxQL
```

Comment:
I feel like this needs more context. (and I've written way too many words here....so, this is more about the concept I'm trying to get across).
There are two ways you can connect Grafana to InfluxDB.  

First, if you have been using Grafana with InfluxDB 1.x and have upgraded to InfluxDB 2.x and want to continue using InfluxQL so your dashboards continue to work....you need to ensure two things have happened:
1) in the upgrade process: the appropriate DBRP mappings were created AND
2) the username/password that you previously used are present in the v1 auth setup....

If you didn't upgrade, but are starting from scratch and simply redirecting new data to a new InfluxDB v2.... you'll still need to check these things.  

Second, you can also use the native 2.0 token-based authentication scheme.  

I think the question here and the guidance we need to provide is around how they decide which path to pursue and what/if anything are the differences.

Reply:
Restructuring this.